tritium-metrics/src/test/java/com/palantir/tritium/metrics/MetricRegistriesTest.java

```diff
@@ -98,6 +98,23 @@ public void testHdrHistogram() {
         assertThat(metrics.timer("timer").getCount()).isEqualTo(1);
     }
 
+    @Test
+    public void testSlidingTimeWindowHistogram() {
+        metrics = MetricRegistries.createWithSlidingTimeWindowReservoirs(1, TimeUnit.MINUTES);
```

Comment:
would be good to test that the observations expire, ideally using a test `Clock` to advance time

Reply:
ack